### PR TITLE
OADP-5261: Update the OADP introduction to include entire cluster backup unsupported

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [https://issues.redhat.com/browse/OADP-5261](https://issues.redhat.com/browse/OADP-5261)

### VERSIONS

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16
* OCP 4.17 → branch/enterprise-4.17
* OCP 4.18 → branch/enterprise-4.18

### Link to docs preview:

* [Introduction to OpenShift API for Data Protection](https://85311--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-intro.html)

### QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/oadp-intro.adoc [hunk 1]
```diff
@@ -9,9 +9,11 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. OADP is also capable of backing up both containerized applications and virtual machines (VMs).
+The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).
 
-However, OADP does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or OpenShift Operators.
+However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.
+
+Nor does {oadp-short} support full cluster backup and restore.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-intro.adoc:16]:
```suggestion
{oadp-short} does not support full cluster backup and restore either.
```

File: backup_and_restore/application_backup_and_restore/oadp-intro.adoc [hunk 2]
```diff
@@ -9,9 +9,11 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. OADP is also capable of backing up both containerized applications and virtual machines (VMs).
+The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).
 
-However, OADP does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or OpenShift Operators.
+However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.
+
+{oadp-short} does not support full cluster backup and restore either.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-intro.adoc:17]:
```suggestion
```

File: backup_and_restore/application_backup_and_restore/oadp-intro.adoc [hunk 3]
```diff
@@ -9,10 +9,11 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. OADP is also capable of backing up both containerized applications and virtual machines (VMs).
+The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).
 
-However, OADP does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or OpenShift Operators.
+However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.
 
+{oadp-short} does not support full cluster backup and restore either.
```
Comment (kaovilai) [inline backup_and_restore/application_backup_and_restore/oadp-intro.adoc:16]:
```suggestion
{oadp-short} support is provided to customer workload namespaces only. Full cluster backup and restore are not supported.
```

**Review by kaovilai — APPROVED** [review]

File: backup_and_restore/application_backup_and_restore/oadp-intro.adoc [hunk 4]
```diff
@@ -9,10 +9,11 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
...
+The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).
 
-However, OADP does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or OpenShift Operators.
+However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.
 
+{oadp-short} support is provided to customer workload namespaces only. Full cluster backup and restore are not supported.
```
Comment (amastbau) [inline backup_and_restore/application_backup_and_restore/oadp-intro.adoc:16]: maybe support is provided to customer workload namespaces, and cluster scope resources. support is provided to customer workload namespaces 
maybe link to the cluster backup and restore from here.

Comment (anarnold97) [reply backup_and_restore/application_backup_and_restore/oadp-intro.adoc]: @amastbau - please review again, most wise and powerful one!

**Review by kaovilai — APPROVED** [review]

**Comment by anarnold97** [comment]
@amastbau - you happy for me to push this to *peer review* please

Thanks

**Review by skopacz1 — APPROVED** [review]
LGTM

**Comment by anarnold97** [comment]
@skopacz1 - thanks

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**